src/components/userModal/UserModal.js

```diff
@@ -0,0 +1,223 @@
+                                        </div>
+                                    </div>
+                                    <div className="divOutside last">
+                                        <div className="mt-3 ml-2 mb-3">
+                                                {this.props.data.preferred_contact_method.map(function(method) {
+                                                    return (
+                                                    <>
+                                                        <img src={tickIcon} alt="Tick" />
+                                                        <span className="info-container-info"> {method}</span>
+                                                        <span className="ml-4"/>
+                                                    </>
+                                                )})}
+                                        </div> 
+                                    </div>
+                                </div>
+                        </Row>
+                        <Row>
+                                <div className="info-container mt-1 rounded mb-3">
+                                    <div className="divOutside">
+                                        <div className="mt-3 ml-2 mb-3">
+                                            <img src={communityIcon} alt="Community"/>
+                                            <span className="info-container-headers"> Jain Community</span>
+                                        </div>
+                                    </div>
+                                    <div className="divOutside last">
+                                        <div className="mt-3 ml-2 mb-3">
+                                                <span className="info-container-info"> {this.props.data.jain_community}</span>
+                                                <Button variant="outline-secondary" className="ml-4">
+                                                    <>
+                                                        <img src={attachmentIcon} alt="Attachment"/>  
+                                                        <span className="info-container-info"> Certificate proof</span>
+                                                    </>
+                                                </Button>
+                                        </div>
+                                    </div>
+                                </div>
+                        </Row>
+                        <Row>
+                            <div className="card border-0 info-container">
+                                    <ToggleButtonGroup type="radio" name="options" value={this.state.status} onChange={this.changeStatus}>
+                                        <ToggleButton value={ACCEPT} className="Btn-accept rounded mb-0">Accept</ToggleButton>
```

Comment:
I guess Figma does not show this. But I think we should hide the "Submit Updates" button initially. When the user clicks on either "Accept" or "Reject", the Submit Updates button should be visible. This would guide user to first click either "Accept" or "Reject" first and then "Submit Updates". Right now, nothing happens on clicking the "Submit Updates" button, which can be confusing for the user.

If the user clicks on "Accept", "Reject" button should be disabled and "Submit Updates" should be enabled.
If the user clicks on "Reject", "Accept" button should be disabled and "Reject Reason" field and "Submit Updates" button should be enabled.

Reply:
So regarding clicking "Submit Updates", if you click on it without selecting any option, an alert comes up to select a valid option
<img width="1440" alt="Screen Shot 2020-08-29 at 4 31 38 PM" src="https://user-images.githubusercontent.com/22218692/91647879-2f7aa100-ea15-11ea-98c5-ee23be92939a.png">

Reply:
Yes that is fine. But we basically do not want to show the "Submit Updates" button, till the user does not click "Accept" or "Reject"

To implement this, you could maybe add `pendingAction` to this component's state, initialized to be empty. When user clicks on "Accept" or "Reject" button, `pendingAction` gets that value. And you can render "Submit Updates" button only when `pendingAction` is not empty.

If you find a better approach than this, feel free to implement :)

Reply:
Yeah that would be the easiest way it could be done ! Added that